### ISSUE TITLE
Cannot verify codes in release builds

### ISSUE THREAD

**Issue by michael-markl** [digitalfabrik/entitlementcard#752]
**Describe the bug**

Cannot verify codes in release builds: Instead of a scanner page, a plain gray page shows up.

Interestingly, the scanner page for activating QR codes does work.

**To Reproduce**
1. Build a release APK
2. Click on 'Ausweisen'-Tab
3. Click on "Gültigkeit prüfen"
4. Click on "Weiter" on the info dialog (if it shows up)

**Expected behavior**
A screen should show up that allows to scan a dynamic/static verification code.

**Screenshots**
After clicking on "Gültigkeit prüfen" and after clicking on "Weiter" on the info-dialog, a screen with a gray background is shown. The camera/qr code scanner page does not show up.
![Screenshot_20230125-101705](https://user-images.githubusercontent.com/7025199/214525177-76f6b8ac-e7ab-420d-b589-df73b61c21f0.jpg)

**Smartphone (please complete the following information):**
 - Device: OnePlus 6
 - OS: Android 11
 - Version 2.1.1

**Additional context**
Not sure, whether iOS is also affected.

**Comment by maxammann** [comment]
I guess those loglines are relevant: 01-25 21:21:40.696 18292 18671 I flutter : Another exception was thrown: Instance of 'DiagnosticsProperty<void>'
01-25 21:21:40.696 18292 18671 I flutter : Another exception was thrown: Instance of 'DiagnosticsProperty<void>'

**Comment by maxammann** [comment]
Stacktrace from debug build:

```
======== Exception caught by widgets library =======================================================
The following assertion was thrown while applying parent data.:
Incorrect use of ParentDataWidget.

The ParentDataWidget Expanded(flex: 1) wants to apply ParentData of type FlexParentData to a RenderObject, which has been set up to accept ParentData of incompatible type ParentData.

Usually, this means that the Expanded widget has the wrong ancestor RenderObjectWidget. Typically, Expanded widgets are placed directly inside Flex widgets.
The offending Expanded is currently placed inside a _InkFeatures widget.

The ownership chain for the RenderObject that received the incompatible parent data was:
  Column ← Expanded ← VerificationQrScannerPage ← DefaultTextStyle ← AnimatedDefaultTextStyle ← _InkFeatures-[GlobalKey#f79e1 ink renderer] ← NotificationListener<LayoutChangedNotification> ← PhysicalModel ← AnimatedPhysicalModel ← Material ← ⋯
When the exception was thrown, this was the stack: 
#0      RenderObjectElement._updateParentData.<anonymous closure> (package:flutter/src/widgets/framework.dart:6042:11)
#1      RenderObjectElement._updateParentData (package:flutter/src/widgets/framework.dart:6059:6)
#2      RenderObjectElement.attachRenderObject (package:flutter/src/widgets/framework.dart:6082:7)
#3      RenderObjectElement.mount (package:flutter/src/widgets/framework.dart:5751:5)
#4      MultiChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:6442:11)
...     Normal element mounting (329 frames)
#333    Element.inflateWidget (package:flutter/src/widgets/framework.dart:3863:16)
#334    MultiChildRenderObjectElement.inflateWidget (package:flutter/src/widgets/framework.dart:6435:36)
#335    Element.updateChild (package:flutter/src/widgets/framework.dart:3592:18)
#336    RenderObjectElement.updateChildren (package:flutter/src/widgets/framework.dart:5964:32)
#337    MultiChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6460:17)
#338    Element.updateChild (package:flutter/src/widgets/framework.dart:3570:15)
#339    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:4904:16)
#340    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5050:11)
#341    Element.rebuild (package:flutter/src/widgets/framework.dart:4604:5)
#342    StatefulElement.update (package:flutter/src/widgets/framework.dart:5082:5)
#343    Element.updateChild (package:flutter/src/widgets/framework.dart:3570:15)
#344    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:4904:16)
#345    Element.rebuild (package:flutter/src/widgets/framework.dart:4604:5)
#346    ProxyElement.update (package:flutter/src/widgets/framework.dart:5228:5)
#347    Element.updateChild (package:flutter/src/widgets/framework.dart:3570:15)
#348    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:4904:16)
#349    Element.rebuild (package:flutter/src/widgets/framework.dart:4604:5)
#350    ProxyElement.update (package:flutter/src/widgets/framework.dart:5228:5)
#351    _InheritedNotifierElement.update (package:flutter/src/widgets/inherited_notifier.dart:107:11)
#352    Element.updateChild (package:flutter/src/widgets/framework.dart:3570:15)
#353    SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6307:14)
#354    Element.updateChild (package:flutter/src/widgets/framework.dart:3570:15)
#355    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:4904:16)
#356    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5050:11)
#357    Element.rebuild (package:flutter/src/widgets/framework.dart:4604:5)
#358    StatefulElement.update (package:flutter/src/widgets/framework.dart:5082:5)
#359    Element.updateChild (package:flutter/src/widgets/framework.dart:3570:15)
#360    SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6307:14)
#361    Element.updateChild (package:flutter/src/widgets/framework.dart:3570:15)
#362    SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6307:14)
#363    Element.updateChild (package:flutter/src/widgets/framework.dart:3570:15)
#364    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:4904:16)
#365    Element.rebuild (package:flutter/src/widgets/framework.dart:4604:5)
#366    ProxyElement.update (package:flutter/src/widgets/framework.dart:5228:5)
#367    Element.updateChild (package:flutter/src/widgets/framework.dart:3570:15)
#368    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:4904:16)
#369    StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5050:11)
#370    Element.rebuild (package:flutter/src/widgets/framework.dart:4604:5)
#371    BuildOwner.buildScope (package:flutter/src/widgets/framework.dart:2667:19)
#372    WidgetsBinding.drawFrame (package:flutter/src/widgets/binding.dart:882:21)
#373    RendererBinding._handlePersistentFrameCallback (package:flutter/src/rendering/binding.dart:378:5)
#374    SchedulerBinding._invokeFrameCallback (package:flutter/src/scheduler/binding.dart:1175:15)
#375    SchedulerBinding.handleDrawFrame (package:flutter/src/scheduler/binding.dart:1104:9)
#376    SchedulerBinding._handleDrawFrame (package:flutter/src/scheduler/binding.dart:1015:5)
#377    _invoke (dart:ui/hooks.dart:148:13)
#378    PlatformDispatcher._drawFrame (dart:ui/platform_dispatcher.dart:318:5)
#379    _drawFrame (dart:ui/hooks.dart:115:31)
====================================================================================================

```

**Comment by michael-markl** [comment]
Closed by #765
